crypto/fipsmodule/rand/urandom_test.cc

```diff
@@ -42,6 +42,13 @@
 #define PTRACE_O_TRACESYSGOOD (1)
 #endif
 
+#if defined(AWSLC_FIPS)
```

Comment:
Can you add a few lines of documentation explaining what this actually does?

Reply:
This information is mostly contained in the description of functions and also above the text fixture defining the test instance (`TEST(URandomTest, Test`). Added a bit more to it.